Games/League of Legends/BETA Client/script.js

```diff
@@ -12,50 +12,123 @@ include(["Functions", "Verbs", "vcrun2015"]);
 // Important: 
 // The BETA-Client has to be installed using the launcher.
 // After the upgrade the BETA Client will do the game patching.
+///////////////////////////////////////////////////////////////////
 
 new CustomInstallerScript()
     .name("League of Legends BETA Client")
     .editor("Riot Games")
-    .applicationHomepage("http://euw.leagueoflegends.com/")
+    .applicationHomepage("http://leagueoflegends.com/")
     .author("Plata, feanor12")
     .installationCommand(function(wizard) {
-        return {command: "msiexec", args: ["/i", "C://LoL_tmp/Lol.EUW.msi","APPDIR=C:\\LoL","/q"]};
+        return {command: "msiexec", args: ["/i", "C://LoL_tmp/LoL.msi", "APPDIR=C:\\LoL", "/q"]};
     })
     .category("Games")
     .wineDistribution("staging")
     .wineVersion("2.0")
     .preInstall(function(wine, wizard) {
-        //Download the setup file
+        // Select the region and download the setup file
+        ////////////////////////////////////////////////
+        var regions = ["EU West",
+                       "Latin America North",
+                       "Latin America South",
+                       "Oceania",
+                       "Japan",
+                       "Turkey",
+                       "Brasil",
+                       "EU Nordic & East",
+                       "North America",
+                       "Russia"];
+        var selectedRegion = wizard.menu("Select your region:", regions);
```

Comment:
Don't know about eclipse. But JavaFXApplication should be the correct target. Can we take this as a new issue to [PlayOnLinux/POL-POM-5](https://github.com/PlayOnLinux/POL-POM-5) please.